src/state/process.rs

```diff
@@ -849,7 +852,10 @@ where
             qc_type.clone(),
         )?;
 
-        if epoch_id == self.epoch_id && round > self.round && self.round + FUTURE_ROUND_GAP > round
+        if qc_type == VoteType::Prevote
+            && epoch_id == self.epoch_id
+            && round > self.round
+            && self.round + FUTURE_ROUND_GAP > round
```

Comment:
do not need round gap filter here, because in the same epoch_id, no one can mock a valid aggregate vote.

Reply:
should handle aggregate vote which round > self.round